src/triangulate/quadedge/QuadEdgeSubdivision.cpp

```diff
@@ -421,7 +421,10 @@ class
         Triangle triangle(triEdges[0]->orig().getCoordinate(),
                           triEdges[1]->orig().getCoordinate(), triEdges[2]->orig().getCoordinate());
         Coordinate cc;
-        triangle.circumcentre(cc);
+        if (triangle.isIsoceles())
```

Comment:
@pramsey It looks like this heuristic doesn't work (mea culpa, since it was my suggestion).  We should remove it, I think.